src/web/mjs/connectors/templates/NovelCool.mjs

```diff
@@ -0,0 +1,88 @@
+import Connector from '../../engine/Connector.mjs';
+import Manga from '../../engine/Manga.mjs';
+
+export default class Novelcool extends Connector {
+
+    constructor() {
+        super();
+        super.id = undefined;
+        super.label = undefined;
+        this.tags = undefined;
+        this.url = undefined;
+
+        this.manga_selector = 'div.book-list div.book-info a';
+        this.manga_page_query = '/category/index_{page}.html'; // {page} will get replaced with the current page number
+        this.chapter_selector = 'div.chapter-item-list a';
+        this.page_selector = 'select.sl-page';
```

Comment:
May use something like `div.foo div:first-of-type div.bar select.sl-page option` to get options

Reply:
Yeah, that works.